common/utils/api-utils.js

```diff
@@ -50,10 +53,15 @@ export const get = async (path, { token, parameters } = {}, axiosClient = Operat
       headers: setAuthorizationHeader(token),
       cancelToken: abort.token,
       params: parameters,
+      paramsSerializer: parameters_ => qs.stringify(parameters_, { arrayFormat: 'repeat' }),
```

Comment:
Can you leave a comment about why this is here 🙏

Reply:
You got it 👍🏼